static/js/components/DefaultObservationPlanTable.jsx

```diff
@@ -127,25 +127,25 @@ const DefaultObservationPlanTable = ({
     );
   };
 
-  const renderFilters = (dataIndex) => {
+  const renderGcnEventFilters = (dataIndex) => {
     const default_observation_plan = default_observation_plans[dataIndex];
 
     return (
       <div>
-        {default_observation_plan
-          ? default_observation_plan.payload.filters
+        {default_observation_plan?.filters
+          ? JSON.stringify(default_observation_plan.filters)
           : ""}
       </div>
     );
   };
 
-  const renderEnd_Date = (dataIndex) => {
+  const renderFilters = (dataIndex) => {
     const default_observation_plan = default_observation_plans[dataIndex];
 
     return (
       <div>
         {default_observation_plan
-          ? default_observation_plan.payload.end_date
+          ? default_observation_plan.payload.filters
```

Comment:
You removed the `payload` in line 135 but you use it here? Is that correct?

Reply:
So basically the start_date and end_date don't mean anything in these default plans as they get replaced.